src/tests/EdFi.DataManagementService.Tests.E2E/Features/References/NaturalForeignKeyValidation.feature.disabled

```diff
@@ -0,0 +1,189 @@
+                          "schoolId": 725
+                      },
+                      "sessionReference": {
+                          "schoolId": 999,
+                          "schoolYear": 2022,
+                          "sessionName": "2021-2022 Spring Semester"
+                      }
+                  }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                  {
+                      "detail": "Data validation failed. See 'validationErrors' for details.",
+                      "type": "urn:ed-fi:api:bad-request:data-validation-failed",
+                      "title": "Data Validation Failed",
+                      "status": 400,
+                      "correlationId": null,
+                      "validationErrors": {
+                          "$.schoolReference.schoolId": [
+                              "All values supplied for 'schoolId' must match. Review all references and align the following conflicting values: '725', '999'"
+                          ],
+                          "$.sessionReference.schoolId": [
+                              "All values supplied for 'schoolId' must match. Review all references and align the following conflicting values: '725', '999'"
+                          ]
+                      }
+                  }
+                  """
+
+        @ignore
+        Scenario: 04 Verify clients cannot create a resource that is missing a reference
+             When a POST request is made to "ed-fi/courseOfferings" with
+                  """
+                  {
+                      "localCourseCode": "ALG-1 TEST-101",
+                      "courseReference": {
+                          "courseCode": "ALG-1",
+                          "educationOrganizationId": 255901001
+                      },
+                      "sessionReference": {
+                          "schoolId": 255901001,
+                          "schoolYear": 2022,
+                          "sessionName": "2021-2022 Spring Semester"
+                      }
+                  }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                  {
+                      "detail": "Data validation failed. See 'validationErrors' for details.",
+                      "type": "urn:ed-fi:api:bad-request:data-validation-failed",
+                      "title": "Data Validation Failed",
+                      "status": 400,
+                      "correlationId": null,
+                      "validationErrors": {
+                          "$.schoolReference": [
+                              "SchoolReference is required."
+                          ]
+                      }
+                  }
+                  """
+
+        @ignore
+        Scenario: 05 Verify clients cannot update a resource that is missing a reference
```

Comment:
Isn't this scenario already covered in `InvalidReferenceValidation.feature`?

Reply:
As per our conversation, this scenario was moved to InvalidReferenceValidation.feature file, because it makes for sense for this scenario to be in this file